tgui/packages/tgui/styles/interfaces/CrewManifest.scss

```diff
@@ -7,7 +7,7 @@ $department_map: (
   'Medical': colors.$teal,
   'Misc': colors.$white,
   'Science': colors.$purple,
-  'Supply': colors.$brown,
+  'Cargo': colors.$brown,
```

Comment:
you are forsaking the alphabetical order btw

Reply:
This is a grievous crime that I have committed